Documentation/Storage-Configuration/Advanced/configuration.md

```diff
@@ -20,11 +20,38 @@ configuration options which users are well advised to consider for any productio
 
 ### Default PG and PGP counts
 
-The number of PGs and PGPs can be configured on a per-pool basis, but it is highly advised to set
-default values that are appropriate for your Ceph cluster. Appropriate values depend on the number
-of OSDs the user expects to have backing each pool. The Ceph [OSD and Pool config
+The number of PGs and PGPs can be configured on a per-pool basis, but it is
+advised to set default values that are appropriate for your Ceph
+cluster.
+Appropriate values depend on the number of OSDs the user expects to have
+backing each pool. These can be configured by declaring pg_num and pgp_num
+parameters under CephBlockPool resource.
+
+For determining the right value for pg_num please refer [placement group
+sizing](ceph-configuration.md#placement-group-sizing)
+
+An example configuration for a cluster of 100 OSDs requiring 100 target PGs per
+OSD with 5% data utilization would look like:
```

Comment:
The 128 PGs for 100 OSDs was based on the example in the pg calc tool that assumed a bunch of other tools? That tool has a lot of defaults that won't make sense for most clusters. I think the point of this example is just to show the yaml for how the desired PGs can be set, so how about simplify this paragraph to the following:
```
In this example configuration, 128 PGs are applied to the pool:
```

Reply:
makes sense! updated, thanks!